webapp-django/crashstats/crashstats/models.py

```diff
@@ -44,17 +47,37 @@ class DeprecatedModelError(DeprecationWarning):
 
 
 def config_from_configman():
-    return configuration(
-        definition_source=[
-            ElasticsearchConfig.required_config,
-            PostgreSQLCrashStorage.required_config,
-            # This required_config defines the logger aggregate
-            socorro_app.App.required_config,
-        ],
+    definition_source = Namespace()
+    definition_source.namespace('logging')
+    definition_source.logging = socorro_app.App.required_config.logging
+
+    definition_source.namespace('elasticsearch')
+    definition_source.elasticsearch.add_option(
+        'elasticsearch_class',
+        default=ElasticsearchConfig,
+    )
+    definition_source.namespace('database')
+    definition_source.database.add_option(
+        'database_storage_class',
+        default=PostgreSQLCrashStorage,
+    )
+    definition_source.namespace('queuing')
+    definition_source.queuing.add_option(
+        'rabbitmq',
```

Comment:
That's quite vague. Why not something more precise like ``rabbitmq_reprocessing_class``? When I see just ``rabbitmq`` I expect a ConnectionContext.